OpenRA.Mods.Common/AI/SupportPowerDecision.cs

```diff
@@ -94,7 +94,7 @@ public int GetAttractiveness(IEnumerable<Actor> actors, Player firedBy)
 		/// <summary>Makes up part of a decision, describing how to evaluate a target.</summary>
 		public class Consideration
 		{
-			public enum DecisionMetric { Health, Value, None }
+			public enum DecisionMetric { Health, Value, BaseBuilding, None }
```

Comment:
Question is can a nuke (keep in mind that this code is used by AIs across mods, so that includes Ion Cannons, Death Hand missile, etc) take out a base building (ConYard in most cases)? Because if not, then that would be a completely wasted super weapon.